proprietary/tokens/rollup.config.mjs

```diff
@@ -2,7 +2,7 @@ import postcss from 'rollup-plugin-postcss';
 import discardDuplicates from 'postcss-discard-duplicates';
 
 export default {
-  input: 'src/theme/index.scss',
+  input: 'src/index.scss',
```

Comment:
Er zat een verschil tussen die twee bestanden. Eentje heeft `:root { <tokens> }` en de andere `.denhaag-theme { <tokens> }`. Is het bewust dat je hier de andere gebruikt?